index.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+const Executor = require('screwdriver-executor-base');
+const path = require('path');
+const request = require('request');
+const jenkins = require('jenkins');
+const fs = require('fs');
+const async = require('async');
+
+class J5sExecutor extends Executor {
+
+    /**
+     * Constructor
+     * @method constructor
+     * @param  {Object} options           Configuration options
+     * @param  {Object} options.host      Jenkins hostname to make requests to
+     * @param  {Object} options.username  Jenkins username
+     * @param  {Object} options.password  Jenkins password/token
+     */
+    constructor(options) {
+        super();
+        this.host = options.host;
+        this.username = options.username;
+        this.password = options.password;
+        // need to pass port nubmer in the future
+        this.crumbUrl =
+    `http://${this.username}:${this.password}@${this.host}:8080/crumbIssuer/api/json`;
```

Comment:
maybe set `http://${this.username}:${this.password}@${this.host}:8080` as baseUrl since you're reusing this later